sys-apps/xinetd/xinetd-2.3.15.recipe

```diff
@@ -0,0 +1,70 @@
+SUMMARY="A better inetd replacement"
+DESCRIPTION="Xinetd is a powerful inetd replacement. Xinetd has access \
+control mechanisms, extensive logging capabilities, the ability to make \
+services available based on time, can place limits on the number of servers \
+that can be started, and has a configurable defence mechanism to protect \
+against port scanners, among other things."
+HOMEPAGE="https://github.com/xinetd-org/xinetd"
+COPYRIGHT="1992 by Panagiotis Tsirigotis
+	1998-2001 by Rob Braun
+	2001-2003 Steve Grubb"
+LICENSE="Xinetd"
+REVISION="1"
+SOURCE_URI="https://github.com/xinetd-org/xinetd/archive/xinetd-${portVersion//\./-}.tar.gz"
+CHECKSUM_SHA256="620b25f4ab4d72fdf32b13797156ea40df2049f1c07e640177e5fec544e9a94c"
+SOURCE_DIR="xinetd-xinetd-${portVersion//\./-}"
+PATCHES="xinetd-$portVersion.patch"
+
+ARCHITECTURES="?x86_gcc2 ?x86_64"
```

Comment:
```diff
-ARCHITECTURES="?x86_gcc2 ?x86_64"
+ARCHITECTURES="x86_gcc2 x86_64"
```

Reply:
Should we enable? Still some old release (and no updates) ...